packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -15,6 +15,8 @@ import { useLocation } from '@reach/router';
 import useQueryParams from '../hooks/useQueryParams';
 import useKeyPress from '../hooks/useKeyPress';
 import { rgba } from 'polished';
+import { useTreatments, useTrack } from '@splitsoftware/splitio-react';
```

Comment:
what do these functions do? I tried looking into the splitIO docs but I couldn't find anything

Reply:
The split docs are a bit difficult to navigate. I actually had to pop open the source code to understand `useTreatments`. Here is the docs for `useTrack`: https://help.split.io/hc/en-us/articles/360038825091-React-SDK#track

Essentially this just loads the configuration for a particular "treatment" on a split and gives you the configuration for it. For example, our `developer-website_global-header-gh-buttons` split has 2 treatments: `on` and `off`. This will load whether to show the `on` or `off` treatment a particular user.